app/enterprise/0.34-x/developer-portal/how-to/serve-assets/aws-cloudfront.md

```diff
@@ -0,0 +1,69 @@
+---
+title: How to Use AWS Cloudfront to Serve Kong Dev Portal Assets
+toc: false
+---
+
+### Introduction
+
+Amazon CloudFront is a fast content delivery network (CDN) service that securely
+ delivers data, videos, applications, and APIs to customers globally with low 
+ latency, high transfer speeds.
+
+In this guide we will create an Amazon S3 bucket, and an Amazon CloudFront 
+distribution to serve static assets for your Kong Developer Portal.
+
+
+### Prerequisites
+
+AWS Account Credentials (with permissions to create S3 buckets, and CloudFront distributions)
+
+
+## Step 1 - Create a new Amazon S3 Bucket
+
+![Create New Bucket](https://s3.us-east-2.amazonaws.com/kong-doc-assets/developer-portal/how-to/aws-cloudfront/aws-bucket.png)
+
+## Step 2 - Upload an asset to the S3 bucket using "Standard" storage class
+
+![Upload Image](https://s3.us-east-2.amazonaws.com/kong-doc-assets/developer-portal/how-to/aws-cloudfront/upload-image.png)
+
+
+## Step 3 - Create a new Amazon CloudFront Distribution
+
+![Create Distribution](https://s3.us-east-2.amazonaws.com/kong-doc-assets/developer-portal/how-to/aws-cloudfront/creat-distribution.png)
+
+## Step 4 - Select the S3 Bucket as your origin
+
+![Select Origin](https://s3.us-east-2.amazonaws.com/kong-doc-assets/developer-portal/how-to/aws-cloudfront/origin-settings.png)
+
+## Step 5 - Restrict the S3 Bucket to only be read by this CloudFront distribution
+
+![Restrict Access](https://s3.us-east-2.amazonaws.com/kong-doc-assets/developer-portal/how-to/aws-cloudfront/restrict-access.png)
+
+## Step 6 - Enabled GZIP 
+
+![Enable GZIP](https://s3.us-east-2.amazonaws.com/kong-doc-assets/developer-portal/how-to/aws-cloudfront/enable-gzip.png)
+
+## Step 7 - Save and wait for the CloudFront distribution to be ready
+
+![Save](https://s3.us-east-2.amazonaws.com/kong-doc-assets/developer-portal/how-to/aws-cloudfront/save-distribution.png)
+
+## Step 9 - Make your Images Public
+
+Once your distribution is ready, find the image you want to use in your S3 Bucket and make it accessible publically.
+
+![Make Images Public](https://s3.us-east-2.amazonaws.com/kong-doc-assets/developer-portal/how-to/aws-cloudfront/make-public.png)
```

Comment:
We should not be encouraging our users to insecurely expose their S3 bucket assets, this is already covered in Step 5, by allowing CloudFront access to S3 bucket assets we get a proper chain of authority.

```User -> CloudFront -> S3```

Following this step, the current flow is possible and you lose out on any CDN enhancements, and potentially a security exploit:

```User -> S3```

Reply:
hmm interesting -- I was not able to get passed the "Access Denied" without making the image public.